swagger.yaml

```diff
@@ -0,0 +1,168 @@
+swagger: "2.0"
+info:
+  version: "1.0"
+  title: My tutorial project
+  description: An app to manage Tutorials
+host: localhost:3000
+basePath: /
+tags:
+  - name: Tutorials
+    description: API for Tutorials in the system
+schemes:
+  - http
+consumes:
+  - application/json
+produces:
+  - application/json
+paths:
+  /tutorials:
+    get:
+      tags:
+        - Tutorials
+      summary: Get all tutorials in system
+      responses:
+        "200":
+          description: OK
+          schema:
+            $ref: "#/definitions/Tutorial"
+        "404":
+          description: "Not found"
+          schema:
+            $ref: "#/definitions/404Response"
+        "500":
+          description: "Internal Server Error"
+          schema:
+            $ref: "#/definitions/500Response"
+
+    post:
+      summary: Create tutorials in system
+      tags:
+        - Tutorials
+      parameters:
+        - name: Tutorials
+          in: body
+          description: User that we want to create
+          schema:
+            $ref: "#/definitions/Tutorial"
```

Comment:
While creating new Tutorial, body only needs to title and description nothing else. so we can't reference `#/definitions/Tutorial` here. We need to create another definition for example `TutorialCreateBody` with only title and description fields.

CC: @JainishaParmar @ashishkava